examples/create-react-app-material-ui/src/registerServiceWorker.js

```diff
@@ -0,0 +1,102 @@
+// In production, we register a service worker to serve assets from local cache.
+
+// This lets the app load faster on subsequent visits in production, and gives
+// it offline capabilities. However, it also means that developers (and users)
+// will only see deployed updates on the "N+1" visit to a page, since previously
+// cached resources are updated in the background.
+
+// To learn more about the benefits of this model, read https://goo.gl/KwvDNy.
+// This link also includes instructions on opting out of this behavior.
+/* @flow */
```

Comment:
I added `/* eslint-disable */` at the start of the file to remove any kind of linting, I will keep the file in its original state. Although I am still getting these 4 errors.

```
\flow is still initializing; this can take some time. [merging inference] |The flow server is not responding (3 retries remaining): [processing] /node_modules/kefir/dist/kefir.js.flow:40
 40:     next?: (value: V) => any,
                        ^ V. invariant position (expected `V` to occur only covariantly)

node_modules/kefir/dist/kefir.js.flow:41
 41:     error?: (error: E) => any,
                         ^ E. invariant position (expected `E` to occur only covariantly)

node_modules/kefir/kefir.js.flow:40
 40:     next?: (value: V) => any,
                        ^ V. invariant position (expected `V` to occur only covariantly)

node_modules/kefir/kefir.js.flow:41
 41:     error?: (error: E) => any,
                         ^ E. invariant position (expected `E` to occur only covariantly)


Found 4 errors

```